### PR TITLE
CIV-5452 Lint errors/warnings

### PR DESCRIPTION
**Pull request by ElifDenizEsea**
**Before creating a pull request make sure that:**

- [x] commit messages are meaningful and follow good commit message guidelines
- [x] README and other documentation has been updated / added (if needed)
- [x] tests have been updated / new tests has been added (if needed)

### JIRA link ###

https://tools.hmcts.net/jira/browse/CIV-5452

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

**Review by paul-pearson — COMMENTED** [review]
It's great to get rid of all these warnings.  Just some whitespace which could be tidied up.

File: src/main/services/features/caseProgression/expertContentBuilder.ts [hunk 1]
```diff
@@ -24,7 +24,7 @@ export const buildExpertReportSection = (section: ExpertSection = null, index =
     .addTitle('PAGES.UPLOAD_DOCUMENTS.EXPERT.EXPERT_REPORT')
     .addInputArray('PAGES.UPLOAD_DOCUMENTS.EXPERT.EXPERT_NAME', '', '', expertReport, 'expertName', section?.expertName, index, form?.errorFor(`${errorFieldNamePrefix}[expertName]`, expertReport))
     .addInputArray('PAGES.UPLOAD_DOCUMENTS.EXPERT.FIELD_EXPERTISE', '', '', expertReport, 'fieldOfExpertise', section?.fieldOfExpertise, index, form?.errorFor(`${errorFieldNamePrefix}[fieldOfExpertise]`, expertReport))
-    .addDateArray('PAGES.UPLOAD_DOCUMENTS.EXPERT.DATE_REPORT_WAS', invalidDateErrors,'PAGES.UPLOAD_DOCUMENTS.DATE_EXAMPLE', expertReport, 'dateInputFields', 'date', section?.dateInputFields?.dateDay.toString(), section?.dateInputFields?.dateMonth.toString(), section?.dateInputFields?.dateYear.toString(), index)
+    .addDateArray('PAGES.UPLOAD_DOCUMENTS.EXPERT.DATE_REPORT_WAS', invalidDateErrors,'PAGES.UPLOAD_DOCUMENTS.DATE_EXAMPLE', expertReport, 'date', section?.dateInputFields?.dateDay.toString(), section?.dateInputFields?.dateMonth.toString(), section?.dateInputFields?.dateYear.toString(), index,'dateInputFields')
```
[the remark this answers is not in the document]
Comment (ElifDenizEsea) [reply src/main/services/features/caseProgression/expertContentBuilder.ts:27]: done

File: src/main/services/features/caseProgression/hearingFee/hearingFeeService.ts [hunk 2]
```diff
@@ -3,12 +3,14 @@ import {CaseEvent} from 'models/events/caseEvent';
...
+import {AppRequest} from 'models/AppRequest';
+import {Request} from 'express';
 const civilServiceApiBaseUrl = config.get<string>('services.civilService.url');
 const civilServiceClient: CivilServiceClient = new CivilServiceClient(civilServiceApiBaseUrl);
 
-export const triggerNotifyEvent = async (claimId: string, req:any, claim: Claim): Promise<void> => {
+export const triggerNotifyEvent = async (claimId: string, req:AppRequest | Request, claim: Claim): Promise<void> => {
   const ccdClaim = translateReferenceNumberToCCD(claim);
-  await civilServiceClient.submitEvent(CaseEvent.APPLY_HELP_WITH_HEARING_FEE, claimId, ccdClaim, req);
+  await civilServiceClient.submitEvent(CaseEvent.APPLY_HELP_WITH_HEARING_FEE, claimId, ccdClaim,<AppRequest> req);
```
[the remark this answers is not in the document]
Comment (ElifDenizEsea) [reply src/main/services/features/caseProgression/hearingFee/hearingFeeService.ts:13]: done

File: src/test/unit/common/models/caseProgression/uploadDocumentsSectionBuilder.test.ts [hunk 3]
```diff
@@ -60,7 +60,7 @@ describe('UploadDocumentsSectionBuilder tests', ()=> {
     //When
     const dateArrayBuilt = new UploadDocumentsSectionBuilder()
       .addDateArray(dateArrayExpected.data.text, errors, dateArrayExpected.data.hint,
-        dateArrayExpected.data.category, dateArrayExpected.data.parentField,  dateArrayExpected.data.field, dateArrayExpected.data.dayValue, dateArrayExpected.data.monthValue, dateArrayExpected.data.yearValue)
+        dateArrayExpected.data.category,  dateArrayExpected.data.field, dateArrayExpected.data.dayValue, dateArrayExpected.data.monthValue, dateArrayExpected.data.yearValue,0,dateArrayExpected.data.parentField)
```
[the remark this answers is not in the document]
Comment (ElifDenizEsea) [reply src/test/unit/common/models/caseProgression/uploadDocumentsSectionBuilder.test.ts]: done

**Review by paul-pearson — APPROVED** [review]

**Review by jarekPierchala — DISMISSED** [review]

**Pull request merged**